cli/tests/unit/flash_test.ts

```diff
@@ -877,8 +833,9 @@ Deno.test(
   },
 );
 
+// TODO(bartlomieju): debug and fix
 Deno.test(
-  { permissions: { net: true } },
+  { permissions: { net: true }, ignore: true },
```

Comment:
@littledivy please take a look

Reply:
It appears there's a problem somewhere in serde_v8 or rusty_v8. The value of `"X-Header-Test": "Æ"` gets converted to a single byte (198).

```
    let v8str = v8::Local::<v8::String>::try_from(value)
      .map_err(|_| Error::ExpectedString)?;
    if !v8str.contains_only_onebyte() {
      return Err(Error::ExpectedLatin1);
    }
```
This code in `serde_v8` passes fine - the `contains_only_onebyte` returns `true` from V8.

Reply:
It appears, the test was wrong. It was using `UTF-8` decoding, but the headers are Latin-1. Updated the test